packages/lesswrong/components/posts/PostsEditForm.tsx

```diff
@@ -16,6 +16,10 @@ import { userIsPodcaster } from '../../lib/vulcan-users/permissions';
 import { SHARE_POPUP_QUERY_PARAM } from './PostsPage/PostsPage';
 import { isEAForum } from '../../lib/instanceSettings';
 import { DynamicTableOfContentsContext } from './TableOfContents/DynamicTableOfContents';
+import { Editor } from '@ckeditor/ckeditor5-core';
+
+const editors : {[doc: string]: Editor} = {}
```

Comment:
Prefer to define record types as `Record<string, Editor>`

Reply:
Also I am not totally clear on what this is doing with respect to `EditorsContext`.  I think we usually handle contexts differently from what I'm seeing here?

Reply:
Happy to talk in person; don't understand the comment just from text